docs/modules/ROOT/pages/index.adoc

```diff
@@ -1257,3 +1231,8 @@ You can push values in the Flash scope in an endpoint using the `flash(name, val
 You can read values from the Flash scope in your Qute views using the `{flash:name}` namespace.
 
 The Flash scope only lasts from one request to the next and is cleared at each request.
+
+[[extension-configuration-reference]]
+== Extension Configuration Reference
+
+include::includes/quarkus-renarde.adoc[opts=optional]
```

Comment:
This is wired in case you have Quarkus config. It will then be automatically copied in `./includes/` after generation and included there (provided your config root is `renarde`).

If you don't have any Quarkus config, just drop this.